src/main/java/gregtech/common/misc/spaceprojects/SpaceProjectWorldSavedData.java

```diff
@@ -99,21 +97,38 @@ public SpaceProjectWorldSavedData(String aData) {
 
     @Override
     public void readFromNBT(NBTTagCompound aNBT) {
-        try (JsonReader reader = new JsonReader(new FileReader(teamProjectsFile))) {
-            spaceTeamProjects = GSON_SPACE_PROJECT.fromJson(reader, spaceTeamProjects.getClass());
-        } catch (Exception e) {
-            System.out.print("FAILED TO LOAD: " + SPACE_TEAM_PROJECTS_JSON);
-            e.printStackTrace();
+        if (!aNBT.hasKey("spaceTeamProjects")) {
+            // We don't have a key? Try to read from file
+            try (JsonReader reader = new JsonReader(new FileReader(teamProjectsFile))) {
+                spaceTeamProjects = GSON_SPACE_PROJECT.fromJson(reader, spaceTeamProjects.getClass());
+            } catch (Exception e) {
+                spaceTeamProjects = null;
+            }
+        }
+        if (spaceTeamProjects == null) {
+            spaceTeamProjects = new HashMap<>();
+        }
+        if (spaceTeamProjects == null || aNBT.hasKey("spaceTeamProjects")) {
```

Comment:
```suggestion
        if (aNBT.hasKey("spaceTeamProjects")) {
```
it will never be null due to the line above